kaniko/code/buildpackconfig/buildPackConfig.go

```diff
@@ -5,33 +5,33 @@ import (
 	"compress/gzip"
 	"encoding/json"
 	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
 	"github.com/GoogleContainerTools/kaniko/pkg/config"
 	"github.com/GoogleContainerTools/kaniko/pkg/dockerfile"
 	"github.com/GoogleContainerTools/kaniko/pkg/executor"
-	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	image_util "github.com/GoogleContainerTools/kaniko/pkg/image"
 	fs_util "github.com/GoogleContainerTools/kaniko/pkg/util"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+	"github.com/sirupsen/logrus"
+
 	"github.com/redhat-buildpacks/poc/kaniko/store"
 	"github.com/redhat-buildpacks/poc/kaniko/util"
-	"github.com/sirupsen/logrus"
-	"io"
-	"io/ioutil"
-	"os"
-	"path"
-	"path/filepath"
-	"strings"
 )
 
 const (
 	homeDir                   = "/"
 	kanikoDir                 = "/kaniko"
-	cacheDir                  = "/cache"
+	cacheDir                  = "/layers/kaniko"
```

Comment:
Is this path customizable using Lifecycle ? If yes, then we should define an ENV var and externalize it